<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quest Room Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>Порт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxPort"/>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxPortStart">
        <property name="text">
         <string>Начать работу</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelBaudRate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Скорость обмена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxBaudRate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>9 600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14 400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19 200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28 800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>38 400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>57 600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>76 800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115 200</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelAddress">
        <property name="text">
         <string>Адрес</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxAddress">
        <item>
         <property name="text">
          <string>0 (молчуны)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Всем</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelErrorResult">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonHello">
        <property name="text">
         <string>Hello</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxTimer">
        <property name="text">
         <string>Опрашивать состояние по таймеру</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labelLEDs">
        <property name="text">
         <string>Светодиоды</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelSmartLEDs">
        <property name="text">
         <string>Умные светодиоды</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBoxSmartLedOne">
        <property name="text">
         <string>включать по одному</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="labelRelays">
        <property name="text">
         <string>Реле</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="5" colspan="2">
       <widget class="QTableView" name="tableViewLeds">
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2" rowspan="5" colspan="2">
       <widget class="QTableView" name="tableViewSmartLeds">
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="checkBoxRelay0">
        <property name="text">
         <string>Реле 1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QCheckBox" name="checkBoxRelay1">
        <property name="text">
         <string>Реле 2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QCheckBox" name="checkBoxRelay2">
        <property name="text">
         <string>Реле 3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QCheckBox" name="checkBoxRelay3">
        <property name="text">
         <string>Реле 4</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelKeys">
        <property name="text">
         <string>Кнопки</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="labelSliders">
        <property name="text">
         <string>Слайдеры</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="labelEncoders">
        <property name="text">
         <string>Энкодеры</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="labelSensors">
        <property name="text">
         <string>Сенсоры</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLabel" name="labelStikyKeys">
        <property name="text">
         <string>Залипшие кнопки</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QListWidget" name="listKeys"/>
      </item>
      <item row="7" column="1">
       <widget class="QListWidget" name="listSliders"/>
      </item>
      <item row="7" column="2">
       <widget class="QListWidget" name="listEncoders"/>
      </item>
      <item row="7" column="3">
       <widget class="QListWidget" name="listSensors"/>
      </item>
      <item row="7" column="4">
       <widget class="QListWidget" name="listStikyKeys"/>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="pushButtonKeys">
        <property name="text">
         <string>Кнопки</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="pushButtonSliders">
        <property name="text">
         <string>Слайдеры</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="pushButtonEncoders">
        <property name="text">
         <string>Энкодеры</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="pushButtonSensors">
        <property name="text">
         <string>Сенсоры</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QPushButton" name="pushButtonStikyKeys">
        <property name="text">
         <string>Залипшие кнопки</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="5">
       <widget class="QPushButton" name="pushButtonState">
        <property name="text">
         <string>Всё состояние разом</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
